frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -45,15 +45,19 @@ const GitSection: React.FC<GitSectionProps> = ({ showSample }) => {
     const url = sampleRepo;
     const ref = getSampleRef(tag);
     const dir = getSampleContextDir(tag);
+    const gitType = detectGitType(url);
     const name = values.name || values.image.selected;
     values.name !== name && setFieldValue('name', name);
     !values.application.name && setFieldValue('application.name', `${name}-app`);
     setFieldValue('git.url', url);
     setFieldValue('git.dir', dir);
     setFieldValue('git.ref', ref);
+    setFieldValue('git.type', gitType);
+    setFieldTouched('git.url', true);
```

Comment:
These above changes are added to address an existing bug in dev catalog flow (using Try sample option)

![screenshot-localhost-9000-2019 10 10-12-06-18 (2)](https://user-images.githubusercontent.com/9964343/66563973-903e8800-eb7c-11e9-9afe-0adfdfeca1f3.png)